operators/constellation-node-operator/api/v1alpha1/scalinggroup_types.go

```diff
@@ -21,10 +26,22 @@ type ScalingGroupSpec struct {
 	NodeImage string `json:"nodeImage,omitempty"`
 	// GroupID is the CSP specific, canonical identifier of a scaling group.
 	GroupID string `json:"groupId,omitempty"`
+	// AutoscalerGroupName is name that is expected by the autoscaler.
+	AutoscalerGroupName string `json:"autoscalerGroupName,omitempty"`
 	// Autoscaling specifies wether the scaling group should automatically scale using the cluster-autoscaler.
 	Autoscaling bool `json:"autoscaling,omitempty"`
+	// Min is the minimum number of nodes in the scaling group.
+	Min int32 `json:"min,omitempty"`
+	// Max is the maximum number of nodes in the scaling group.
+	Max int32 `json:"max,omitempty"`
```

Comment:
```suggestion
	// Min is the minimum number of nodes in the scaling group (used by cluster-autoscaler).
	Min int32 `json:"min,omitempty"`
	// Max is the maximum number of autoscaled nodes in the scaling group (used by cluster-autoscaler).
	Max int32 `json:"max,omitempty"`
```

This or similar description to distinguish the from actual min / max in scaling groups.